lib/content-services/src/lib/content-node-selector/content-node-selector.module.ts

```diff
@@ -31,31 +29,28 @@ import { NameLocationCellComponent } from './name-location-cell/name-location-ce
 import { UploadModule } from '../upload/upload.module';
 import { SearchQueryBuilderService } from '../search/services/search-query-builder.service';
 import { ContentDirectiveModule } from '../directives/content-directive.module';
+import { DropdownSitesComponent } from './site-dropdown/sites-dropdown.component';
+
+export const CONTENT_NODE_SELECTOR_DIRECTIVES = [ContentNodeSelectorPanelComponent, NameLocationCellComponent, ContentNodeSelectorComponent];
 
+/** @deprecated import `CONTENT_NODE_SELECTOR_DIRECTIVES` or standalone components directly  */
 @NgModule({
     imports: [
         FormsModule,
         ReactiveFormsModule,
         CoreModule,
         CommonModule,
         MaterialModule,
-        SitesDropdownModule,
+        DropdownSitesComponent,
         BreadcrumbModule,
         SearchModule,
         DocumentListModule,
         UploadModule,
-        ContentDirectiveModule
-    ],
-    exports: [
-        ContentNodeSelectorPanelComponent,
-        NameLocationCellComponent,
-        ContentNodeSelectorComponent
-    ],
-    declarations: [
-        ContentNodeSelectorPanelComponent,
-        NameLocationCellComponent,
-        ContentNodeSelectorComponent
+        ContentDirectiveModule,
+        NameLocationCellComponent
     ],
+    exports: [ContentNodeSelectorPanelComponent, NameLocationCellComponent, ContentNodeSelectorComponent],
+    declarations: [ContentNodeSelectorPanelComponent, ContentNodeSelectorComponent],
```

Comment:
Should we maybe have 
```
exports: [CONTENT_NODE_SELECTOR_DIRECTIVES],
declarations: [CONTENT_NODE_SELECTOR_DIRECTIVES],
```
Not much of a functional change, but would be nice to have for consistency.
Also, if not, can we confirm if the missing entry for NameLocationCellComponent in declarations is intentional?

Reply:
those are not yet fully migrated to standalone, I will do that in another jira that migrates the rest of the dialog